bundler/lib/bundler/ui/json_shell.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module Bundler
+  module UI
+    class JsonShell < UI::Shell
+      def table(header, rows, options = {})
+        return unless info?
+        empty_row = Hash[header.keys.map {|k| [k, nil] }]
```

Comment:
Avoid one intermediate Array.

```suggestion
        empty_row = header.keys.to_h {|k| [k, nil] }
```